test/test-api-wmata.js

```diff
@@ -0,0 +1,41 @@
+require('dotenv').config();
+var path = require('path');
+var tape = require('tape-expect');
+var tapeNock = require('tape-nock');
+var wmataApi = require('../lib/api-wmata.js');
+var props = require('../resources/properties.json');
+
+var test = tapeNock(tape, {
+  fixtures: path.join(__dirname, 'fixtures')
+});
+
+test('wmata api get station prediction valid key', function (t) {
+  var url = props.stationArrivalWmataUrl + 'B02';
+  wmataApi.get(url, function (response) {
+    t.plan(2);
+    t.ok(response);
+    t.expect(response.Trains[0]).to.have.keys('Car', 'Destination', 'DestinationCode', 'DestinationName', 'Group',
+      'Line', 'LocationCode', 'LocationName', 'Min');
+    t.end();
+  });
+});
+
+test('wmata api get station prediction invalid key', function (t) {
```

Comment:
This test isn't testing our code. It is testing the external API. I think it should be removed.